gdc_client/common/config.py

```diff
@@ -40,7 +40,7 @@ class GDCClientConfigShared(object):
         "n_processes": ConfigParser.getint,
         "retry_amount": ConfigParser.getint,
         "wait_time": ConfigParser.getfloat,
-        "no_segment_md5sum": ConfigParser.getboolean,
+        "no_segment_md5sums": ConfigParser.getboolean,
```

Comment:
If we're changing and adding configurable options we should make a note to let user services know.

Reply:
so its a flag that was already there but would never work because of this typo, so with this fix, users should be now getting the functionality they thought they were getting